.travis.yml

```diff
@@ -33,13 +35,15 @@ env:
     - CONDA_CHANNEL_PRIORITY=True
     - NUMPY_VERSION=stable
 
+
+
 matrix:
   fast_finish: true
   include:
     - os : linux
       env: NAME='minimal'
            PYTHON_VERSION=2.7
-           SETUP_CMD='--with-memleak'
+           MAIN_CMD='python ./testsuite/MDAnalysisTests/mda_nosetests analysis auxiliary coordinates core formats topology --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak && python ./testsuite/MDAnalysisTests/mda_nosetests utils --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak'
```

Comment:
Looking closer, it cannot be `SETUP_CMD` indeed. But it can be a variable with an other name, like `NOSE_SPECIAL_OPT`. You can assign`NOSE_SPECIAL_OPT` in the individual `env` sections, just before you assign `MAIN_CMD`, and refer to`NOSE_SPECIAL_OPT` while assigning `MAIN_CMD`. This way you show the difference between the builds; now, the difference is hidden in a very long command line.

Also, the idea you had to have all the nose option in a global variable makes much more sense now. If you repeat the nose command for all the builds, you want to avoid duplication as much as possible to avoid mistakes (e.g. an option is changed in some builds but not all of them).

About `&&`, I may not have been clear enough but I am saying the same thing in both comments. We want all tests to run, regardless of the passing status of the previous ones. `&&` means "and"; `cmd1 && cmd2` means "run `cmd1`, and if it succeeded then run `cmd2`". What we want is "run `cmd1`, run `cmd2`"; this is said with `cmd1; cmd2`.

All in all, I would see something in the lines of:

```
MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests ${NOSE_OPT} ${NOSE_SPECIAL_OPT} ${RUN1_TESTS}; python ./testsuite/MDAnalysisTests/mda_nosetests ${NOSE_OPT} ${NOSE_SPECIAL_OPT} ${RUN2_TESTS}"
```